packages/mobile/src/web3/reducer.ts

```diff
@@ -1,11 +1,10 @@
 import { getRehydratePayload, REHYDRATE, RehydrateAction } from 'src/redux/persist-helper'
-import { Actions, ActionTypes, UpdateWeb3SyncProgressAction } from 'src/web3/actions'
+import { Actions, ActionTypes } from 'src/web3/actions'
 
 export interface State {
-  syncProgress: number
-  syncProgressData: {
+  syncProgress: {
+    startingBlock: number
```

Comment:
@martinvol I aligned this to the names web3 uses, which you also use in `actions.ts`